submissions/ayokofemi/Database 1/init.sql

```diff
@@ -0,0 +1,44 @@
+--for the foodTable--
+CREATE TABLE food (
+id INT NOT NULL AUTO_INCREMENT,
+type VARCHAR(20) NOT NULL,
+PRIMARY KEY (id)
+);
+
+--for the vendorFood--
+CREATE TABLE vendor_food (
+id INT NOT NULL AUTO_INCREMENT,
+admin_id INT NOT NULL,
+food_id INT NOT NULL,
+name VARCHAR(30) NOT NULL,
+amount FLOAT(4,2) NOT NULL,
+PRIMARY KEY (id)
+);
+
+--for the adminsTable--
+CREATE TABLE admins (
+id INT NOT NULL AUTO_INCREMENT,
+name VARCHAR(30) NOT NULL,
+type  VARCHAR(16) NOT NULL,
+email_address VARCHAR(40) NOT NULL,
+phone_number VARCHAR(18) NOT NULL,
```

Comment:
This should not be VARCHAR, make it INT, so that if an applicant types in capital letter o and not Zero 0. it will be rejected.